uportal-war/src/main/resources/properties/contexts/importExportContext.xml

```diff
@@ -600,20 +615,6 @@
         </property>
     </bean>
 
-    <bean id="export-pags-group" class="org.danann.cernunnos.runtime.PojoTask">
-        <property name="location" value="classpath://org/jasig/portal/io/export-pags-group.crn" />
-        <property name="requestAttributes">
-            <util:map>
-                <entry key="SqlAttributes.DATA_SOURCE" value-ref="PortalDb" />
-                <entry key="SqlAttributes.TRANSACTION_MANAGER" value-ref="transactionManager" />
-                <entry key="isUserIncluded" value-ref="isUserIncluded" />
-                <entry key="jpaPersonAttributesGroupDefinitionDao" value-ref="jpaPersonAttributesGroupDefinitionDao" />
-                <entry key="jpaPersonAttributesGroupTestDefinitionDao" value-ref="jpaPersonAttributesGroupTestDefinitionDao" />
-                <entry key="jpaPersonAttributesGroupTestGroupDefinitionDao" value-ref="jpaPersonAttributesGroupTestGroupDefinitionDao" />
-            </util:map>
-        </property>
-    </bean>
-
```

Comment:
Not at all.

There are 2 approaches in current use for implementing an export;  Shawn provided them both.  I'm deleting the unused (and "older") solution.